viewcontroller/viewcontroller.go

```diff
@@ -0,0 +1,430 @@
+package viewcontroller
```

Comment:
Great work!

If you copied something here please add comment with credit

Reply:
added